src/app/components/tree-page/tree-page.component.ts

```diff
@@ -88,4 +90,8 @@ export class TreePageComponent {
     "children": [],
     "spouse": null
   }`);
+
+  selectNode(e: Event) {
+    console.log({ e });
```

Comment:
Do we need to log the event?

Reply:
Yeah... think it's good here for testing.